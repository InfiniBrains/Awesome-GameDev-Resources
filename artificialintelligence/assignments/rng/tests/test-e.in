5 25 100 999
