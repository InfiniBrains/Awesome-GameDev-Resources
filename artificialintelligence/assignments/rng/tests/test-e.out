145
929
597
535
281
959
603
249
281
777
223
925
905
238
567
771
118
925
469
172
112
780
229
521
263
